lib/bigcommerce/connection.rb

```diff
@@ -39,19 +43,19 @@ def verify_peer=(verify)
     end
 
     def ssl_ca_file=(path)
-      @configuration.ssl_ca_file = path
+      @configuration[:ssl_ca_file] = path
     end
 
     def ssl_client_key=(path,passphrase=nil)
```

Comment:
For this method... I'd prefer to keep the initialize a little cleaner, I think we have two options.

1. We can check to see if the path param is a hash, if it is, we can pull out the path / params.
2. We can create a new method with a new key which will call the old version after pulling out the params.